src/ts/mapboxMap.ts

```diff
@@ -53,7 +60,7 @@ module powerbi.extensibility.visual {
             this.palette = new Palette(this, options.host)
         }
 
-        onUpdate(map, settings, zoom, updatedHandler: Function) {
+        onUpdate(map, settings, zoom, geocoder, updatedHandler: Function) {
```

Comment:
Is the `geocoder` object a required input to onUpdate() here since it's a class variable?